src/udev/meson.build

```diff
@@ -124,12 +124,30 @@ libudev_basic = static_library(
         c_args : ['-fvisibility=default'])
 
 libudev_static = static_library(
-        'udev',
+        'udev_static',
         'udev.h',
         include_directories : includes,
         link_with : udev_link_with,
         link_whole : libudev_basic)
 
+static = get_option('static-libudev')
```

Comment:
Please make the name of the variable match the name of the option, i.e. `static_libudev`. Also the same for the other variable.